test/index.js

```diff
@@ -451,4 +473,79 @@ describe('KeyringController', function () {
       );
     });
   });
+
+  describe('Manifest v3 changes', function () {
+    it('can still verify password', async function () {
+      // Log the user out
+      await keyringController.setLocked();
+
+      // Attempt to verify the password
+      const result = await keyringController.verifyPassword(password);
+      expect(result).toBeTruthy();
+    });
+
+    it('can still unlock with password after being migrated and locked', async function () {
```

Comment:
It's unclear to me. Where did the migration happen?
A `before` call would help me understand what's the precondition, even if redundant.